src/Platform/Microsoft.Testing.Platform/OutputDevice/ProxyOutputDevice.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Testing.Platform.Extensions.OutputDevice;
+using Microsoft.Testing.Platform.Hosts;
+using Microsoft.Testing.Platform.ServerMode;
+
+namespace Microsoft.Testing.Platform.OutputDevice;
+
+// Any interfaces that can have special treatment for any output device should be implemented by
+// this proxy class and be forwarded properly.
+// This is not so good. How can we make sure we are not missing any interfaces that may be implemented by external output devices?
```

Comment:
Comment seems to be outdated and could be dropped.

Reply:
It is. Thanks!

Reply:
Fixed